diff --git a/public/emHdBindings.js b/public/emHdBindings.js
--- a/public/emHdBindings.js
+++ b/public/emHdBindings.js
@@ -1000,5 +921,5 @@
       what = "Aborted(" + what + ")";
       err(what);
-      ABORT = true;
+      // ABORT = true; // this does not allow anything to work after being set, however we're actually okay to try other assets so we shouldn't do this
       EXITSTATUS = 1;
       what += ". Build with -sASSERTIONS for more info.";
